avalon/tools/workfiles/app.py

```diff
@@ -74,6 +74,9 @@ def __init__(self, root):
         self.ok_button.pressed.connect(self.on_ok_pressed)
         self.cancel_button.pressed.connect(self.on_cancel_pressed)
 
+        # Allow "Enter" key to accept the save.
+        self.ok_button.setDefault(True)
+
```

Comment:
I found out that in Houdini, the comment field somehow could not have focus when the dialog pops up.
Maybe we could also add the line
```python
self.comment_lineedit.setFocus()
```
so the UX in both DCC would align :P

Reply:
Resolved with https://github.com/getavalon/core/pull/402/commits/d5c5d3736ba2cafaa81529d1e4b1c379060cd308